docker-compose.yaml

```diff
@@ -0,0 +1,35 @@
+version: "3.2"
+
+services:
+  api-gateway:
+    image: itl-teams/api-gateway:latest
+    build: ./api/example
+    container_name: api-gateway
+    restart: unless-stopped
+    ports:
+      - "80:80"
+
+  php-accountAndSignature:
+    container_name: php-accountAndSignature
+    restart: unless-stopped
+    image: itl-teams/php:latest
+    build: ./services/php-accountAndSignature
+    ports:
+      - "8081:80"
```

Comment:
### Note
Same occurs with ports, you don't need this ports for local dev. Make your request to Proxy server directly.